src/Gzero/Core/Parsers/ArrayParser.php

```diff
@@ -0,0 +1,156 @@
+<?php namespace Gzero\Core\Parsers;
+
+use Gzero\Core\Query\QueryBuilder;
+use Gzero\InvalidArgumentException;
+use Illuminate\Http\Request;
+
+class ArrayParser implements ConditionParser {
+
+    /** @var string */
+    protected $name;
+
+    /** @var string */
+    protected $operation = 'in';
```

Comment:
I would move this assignment of default operation down to the `parse` method.
I would keep or the cases in one place.

Yes... I know it has been written this way before BUT if it's a better approach WHY not to learn and improve?

Reply:
@bartero I'm not sure about that, this is a common practice to define default values here - e.g. `protected $availableOperations = ['in', 'not in'];`

I think that benefit of this approach is that you see all default values right away, instead of finding out about them in some methods.

So I think that this is a better approach than having them in some other place.